### PR TITLE
Added a new theme colour for link hover

### LINKED ISSUE #91: Change hover to a dark blue
## Description
Someone pointed out that the hover effect on links is difficult to read, particularly on mobile

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #91 

## Description

- Added a new theme colour for link hover to make it dark blue

@neontribe/contemplating-action

🐧

### CODE REVIEW

File: src/styles/theme.scss [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
 
 @font-face {
-  font-family: 'Nunito';
+  font-family: "Nunito";
   font-style: normal;
   font-weight: black;
-  src: url('./fonts/nunito-black.eot'); /* IE9 */
-  src: url('./fonts/nunito-black.woff') format('woff'), /* Modern Browsers */
-       url('./fonts/nunito-black.ttf') format('truetype'); /* Mobile */
+  src: url("./fonts/nunito-black.eot"); /* IE9 */
```
Comment (rosejbon) [inline src/styles/theme.scss:23]: It seems to have formatted weirdly here

File: src/styles/theme.scss [hunk 2]
```diff
@@ -2,27 +2,27 @@
...
 @font-face {
   font-family: "Lato";
   font-style: normal;
   font-weight: bold;
-  src: url('./fonts/lato-bold.eot'); /* IE9 */
-  src: url('./fonts/lato-bold.woff') format('woff'), /* Modern Browsers */
-       url('./fonts/lato-bold.ttf') format('truetype'); /* Mobile */
+  src: url("./fonts/lato-bold.eot"); /* IE9 */
+  src: url("./fonts/lato-bold.woff") format("woff"),
+    /* Modern Browsers */ url("./fonts/lato-bold.ttf") format("truetype"); /* Mobile */
```
Comment (rosejbon) [inline src/styles/theme.scss:16]: And here

File: src/styles/theme.scss [hunk 3]
```diff
@@ -115,7 +115,7 @@ h6 {
 
 h1 {
   font-size: 1.75em;
-  font-family: 'Nunito', 'Lato', sans-serif;
+  font-family: "Nunito", "Lato", sans-serif;
```
Comment (rosejbon) [inline src/styles/theme.scss:118]: It seems to have added " instead of ', dont know if that is important

**Comment by HedgehogHolly** [comment]
@roseb04 its just my 'prettifier' trying to make things pretty. It's not made any proper changes.

**Comment by rosejbon** [comment]
> @roseb04 its just my 'prettifier' trying to make things pretty. It's not made any proper changes.

ahhh, that makes sense, in some cases it has made things less pretty, but apart from that looks good

**Review by rosejbon — APPROVED** [review]

**Comment by HedgehogHolly** [comment]
I know! It's an odd one

File: src/styles/theme.scss (hunk 2 above)
Comment (Nikomus) [reply src/styles/theme.scss:16]: The comment should be at the end of the line, it's referring to the woff and woff2 formats.

**Comment by Nikomus** [comment]
A project may have conventions which don't line up with prettier, and this one doesn't explicitly use it. If your editor is running prettier on save then files that have changed in this PR will have new formatting and other files won't. We could raise a ticket to standardise non-elm files and use prettier on everything it knows how to format in this and ca-core.

If any code formatter is doing something unhelpful like moving comments to the wrong line based on line length then in VS Code you can open the command palette `Ctrl+Shift+P` and type "save without formatting", to save changes without the formatter modifying them.

**Pull request merged**